pages/map.tsx

```diff
@@ -81,29 +81,32 @@ export default function Home() {
   const HomeHeader = dynamic(() => import('@/components/Mappage/HomeHeader'))
   const NavBar = dynamic(() => import('@/components/NavBar'))
 
-  //our final return for home
   return (
     <>
       <meta name="viewport" content="width=device-width, initial-scale=1.0"></meta>
       <Head>
-        <title>Create Next App</title>
+        <title>Map tool</title>
```

Comment:
Question: do we want it to say "Map Tool?"
maybe just map, or Discover locations or something